README.md

```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+# Calculator-App
+[Live Link](https://calculator-necmi.herokuapp.com/)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A  ![](https://img.shields.io/badge/Microverse-blueviolet)  React Module Project
 
-## Available Scripts
+A Calculator App is developed.
 
-In the project directory, you can run:
+## Built With
 
-### `yarn start`
+- JavaScript
+- React
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Getting Started
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+To get a local copy up and running follow these simple example steps.
 
-### `yarn test`
+- Clone this project by typing ```git clone git@github.com:necmigunduz/calculator-react.git```
+- cd into the root directory
```

Comment:
The README file should include instructions about how to run the app locally. However, From cloning to cd into the directory makes our objective incomplete. To make it complete, kindly add the following:
- `npm install` - This will install all the packages in the package.json file
- `npm start` - This will run the app locally in the local environment